prep-javascript-variables/index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>JavaScript Variables</title>
+  </head>
+  <body>
+    <script>
+      var fullName = 'James Cody Crawford';
+      console.log(fullName);
+
+      var numberOfPets = 'Two';
+      console.log(numberOfPets);
+
+      var likesVegetables = true > false;
+      console.log(likesVegetables);
+
+      var nothing = 'null';
```

Comment:
Here you should leave out the quotation marks so that JavaScript knows that `nothing` is a `null` value and not a string.